content/docs/zero/import.md

```diff
@@ -0,0 +1,125 @@
+---
+# cSpell:ignore freebind
+id: import
+title: Import Core config to Pomerium Zero
+sidebar_label: Core to Zero Import
+description: Learn how to import an existing Pomerium Core configuration into a Pomerium Zero cluster.
+---
+
+Learn how to import your existing open-source Pomerium Core configuration to Pomerium Zero.
+
+## Overview
+
+The **Core to Zero** import feature enables you to import an existing Pomerium Core configuration into a Pomerium Zero cluster with a simple CLI command. A successful import:
+
+- converts entities in Core to Pomerium Zero.
+- removes duplicate policies from the configuration.
+- prevents partial imports with quota checks.
+
+## Who is this for?
+
+The Core to Zero import feature is for users that want to use Pomerium Zero with their existing Pomerium Core configuration. Without the import capability, you’d have to manually configure each route, policy, certificate, and config setting in Pomerium Zero.
+
+This feature automates the process of importing your existing configuration so you don’t have to.
+
+## Core to Zero import definitions
+
+- **Entity:** any route, policy, certificate, or self-hosted domain defined in a Pomerium configuration.
+- **Replica:** a separate Pomerium configuration that shares a storage backend with other replicas running in the same deployment.
+- **Quota:** the amount of entities and replicas you can define in a cluster for free.
+
+## Supported settings
```

Comment:
What is the value of providing this information? I don't think we want to bother the reader to go and confirm if they are using these configurations, since our import process will tell them later. Maybe this is useful at the bottom of the page as supplemental info?

Reply:
I'll move Quotas and Supported settings to a subsection at the bottom of the page called "Additional resources". Let me know your thoughts here; otherwise, we can remove this section entirely.